contracts/protocols/mainnet/morpho/helpers.sol

```diff
@@ -0,0 +1,369 @@
+    }
+
+    struct MorphoData {
+        MarketDetail[] aaveMarketsCreated;
+        MarketDetail[] compMarketsCreated;
+        bool isClaimRewardsPausedAave;
+        bool isClaimRewardsPausedComp;
+    }
+
+    struct TokenConfig {
+        address poolTokenAddress;
+        address underlyingToken;
+        uint256 decimals;
+    }
+
+    struct AaveMarketDetail {
+        uint256 aEmissionPerSecond;
+        uint256 vEmissionPerSecond;
+        uint256 availableLiquidity;
+        uint256 liquidityRate;
+        uint256 ltv;
+        uint256 liquidationThreshold;
+        uint256 liquidationBonus;
+    }
+
+    struct CompoundMarketDetail {
+        uint256 compSpeed;
+        uint256 compSupplySpeed;
+        uint256 compBorrowSpeed;
+        uint256 collateralFactor;
+    }
+
+    struct MarketDetail {
+        TokenConfig config;
+        uint256 avgSupplyRate; //in wad
+        uint256 avgBorrowRate; //in wad
+        uint256 p2pSupplyRate;
+        uint256 p2pBorrowRate;
+        uint256 poolSupplyRate;
+        uint256 poolBorrowRate;
+        uint256 totalP2PSupply;
+        uint256 totalPoolSupply;
+        uint256 totalP2PBorrows;
+        uint256 totalPoolBorrows;
+        uint256 poolSupplyIndex; //exchange rate of cTokens for compound
+        uint256 poolBorrowIndex;
+        uint256 p2pSupplyDelta; //The total amount of underlying ERC20 tokens supplied through Morpho,
+        //stored as matched peer-to-peer but supplied on the underlying pool
+        uint256 p2pBorrowDelta; //The total amount of underlying ERC20 tokens borrow through Morpho,
+        //stored as matched peer-to-peer but borrowed from the underlying pool
+        uint256 reserveFactor;
+        AaveMarketDetail aaveData;
+        CompoundMarketDetail compData;
+        Flags flags;
+    }
+
+    struct Flags {
+        bool isPaused;
+        bool isPartiallyPaused;
+        bool isP2PDisabled;
+    }
+
+    struct UserMarketData {
+        MarketDetail marketData;
+        uint256 borrowRate;
+        uint256 supplyRate;
+        uint256 totalSupplies;
+        uint256 totalBorrows;
+        uint256 p2pBorrows;
+        uint256 p2pSupplies;
+        uint256 poolBorrows;
+        uint256 poolSupplies;
+        uint256 maxWithdrawable;
+        uint256 maxBorrowable;
+    }
+
+    struct UserData {
+        uint256 healthFactor; //calculated by updating interest accrue indices for all markets for comp
+        uint256 collateralValue; //calculated by updating interest accrue indices for all markets for comp
+        uint256 debtValue; //calculated by updating interest accrue indices for all markets for comp
+        uint256 maxDebtValue; //calculated by updating interest accrue indices for all markets for comp
+        bool isLiquidatable;
+        uint256 liquidationThreshold; //for AAVE
+        uint256 unclaimedRewards; //only for compound as of now
+        UserMarketData[] marketData;
+    }
+
+    IAaveLens internal aavelens = IAaveLens(getAaveLens());
```

Comment:
@Richa-iitr you can directly assign the address here instead of creating a function to just return address

Reply:
Got it updating.

Reply:
Done.